{"schemaVersion":{"minor":3,"patch":0,"major":0},"kind":"article","identifier":{"interfaceLanguage":"swift","url":"doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/UserError\/LocalizedError-Implementations"},"sections":[],"topicSections":[{"generated":true,"title":"Instance Properties","identifiers":["doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/UserError\/failureReason","doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/UserError\/helpAnchor","doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/UserError\/recoverySuggestion"]}],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/rxfireauth\/usererror\/localizederror-implementations"]}],"hierarchy":{"paths":[["doc:\/\/RxFireAuth\/documentation\/RxFireAuth","doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/UserError"]]},"metadata":{"role":"collectionGroup","modules":[{"name":"RxFireAuth"}],"title":"LocalizedError Implementations"},"references":{"doc://RxFireAuth/documentation/RxFireAuth/UserError":{"kind":"symbol","url":"\/documentation\/rxfireauth\/usererror","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"UserError","kind":"identifier"}],"identifier":"doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/UserError","title":"UserError","role":"symbol","type":"topic","navigatorTitle":[{"text":"UserError","kind":"identifier"}],"abstract":[{"type":"text","text":"Errors thrown by  "},{"type":"codeVoice","code":"UserManagerType"},{"text":" implementations.","type":"text"}]},"doc://RxFireAuth/documentation/RxFireAuth":{"abstract":[],"type":"topic","role":"collection","kind":"symbol","identifier":"doc:\/\/RxFireAuth\/documentation\/RxFireAuth","title":"RxFireAuth","url":"\/documentation\/rxfireauth"},"doc://RxFireAuth/documentation/RxFireAuth/UserError/helpAnchor":{"identifier":"doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/UserError\/helpAnchor","abstract":[],"fragments":[{"kind":"keyword","text":"var"},{"text":" ","kind":"text"},{"kind":"identifier","text":"helpAnchor"},{"kind":"text","text":": "},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"kind":"text","text":"?"}],"url":"\/documentation\/rxfireauth\/usererror\/helpanchor","kind":"symbol","title":"helpAnchor","role":"symbol","type":"topic"},"doc://RxFireAuth/documentation/RxFireAuth/UserError/failureReason":{"url":"\/documentation\/rxfireauth\/usererror\/failurereason","title":"failureReason","abstract":[],"role":"symbol","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"text":"failureReason","kind":"identifier"},{"kind":"text","text":": "},{"text":"String","preciseIdentifier":"s:SS","kind":"typeIdentifier"},{"kind":"text","text":"?"}],"type":"topic","identifier":"doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/UserError\/failureReason","kind":"symbol"},"doc://RxFireAuth/documentation/RxFireAuth/UserError/recoverySuggestion":{"identifier":"doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/UserError\/recoverySuggestion","abstract":[],"fragments":[{"kind":"keyword","text":"var"},{"text":" ","kind":"text"},{"kind":"identifier","text":"recoverySuggestion"},{"kind":"text","text":": "},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"},{"kind":"text","text":"?"}],"url":"\/documentation\/rxfireauth\/usererror\/recoverysuggestion","kind":"symbol","title":"recoverySuggestion","role":"symbol","type":"topic"}}}